src/controllers/authenticationController.ts

```diff
@@ -214,6 +226,13 @@ export default class AuthenticationController {
             LoginTypeEnum.LEGACY,
           ];
 
+    // Retrieve user IP from request
+    const errorOrUserIp = IPString.decode(spidUser.getAcsOriginalRequest()?.ip);
+
+    if (isUserElegibleForFastLoginResult && E.isLeft(errorOrUserIp)) {
+      return ResponseErrorInternal("Error reading user IP");
+    }
```

Comment:
We are blocking the login if, for some reason, the IP is invalid or missing. Can we consider this parameter as optional?

Reply:
Right now IP address is required by the new notify-login endpoint.